src/modules/ie-objects/elasticsearch/templates/fuzzy/search-query.json

```diff
@@ -44,7 +59,13 @@
         "schema_creator",
         "schema_spatial_coverage",
         "schema_temporal_coverage",
-        "dcterms_medium"
+        "dcterms_medium",
+        "dcterm_format",
+        "ebucore_object_type",
+        "schema_publisher",
```

Comment:
I see that you only added schema_publisher once and not for all multi-match sections. Perhaps that is on purpose because that field is not mapped as full text?
This is repeated for all multi match sections for the schema_creator field, but it is (for now) also not mapped as full text.

So we best keep both fields the same way.

Bart has expressed intention to make both fields full text, so i think we best anticipate and add both fields to all multi match sections:
* "schema_creator",
* "schema_publisher",

Reply:
Correct that was on purpose based on the mapping found in _mapping endpoint of ES. But I will adjusted according as you mentioned.

Reply:
"schema_creator" already exists in all multi match fields so I'll only be adding "schema_publisher"